docs/reference/manifest/type.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Type element in the manifest file
+description: ''
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# Type element
+
+specifies if the equivalent add-in is a COM addin or an XLL.
+
+**Add-in type:** Task pane, Custom function
+
+## Syntax
+
+```XML
+    <Type>{Add-in type}</Type>  
```

Comment:
Based on [this page](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/allowsnapshot), this should be:
```
<Type> [COM | XLL]</Type>
```